EIPS/eip-1052.md

```diff
@@ -3,7 +3,7 @@ eip: 1052
 title: EXTCODEHASH opcode
 author: Nick Johnson <arachnid@notdot.net>, Paweł Bylica <pawel@ethereum.org>
 discussions-to: https://ethereum-magicians.org/t/extcodehash-opcode/262
-status: Draft
+status: Accepted
```

Comment:
Should be Final

Reply:
I think #1642 supposed to set it to final, according to the "process" :)